algorithm/cgal/SurfaceMeshQuery.test.js

```diff
@@ -153,7 +157,8 @@ test('Clip Segment by Surface', (t) => {
   {
     // Segment along face
     const segments = [];
-    query.clipSegmentApproximate(
+    query.intersectSegmentApproximate(
```

Comment:
What makes these approximate?

Reply:
The segments are specified using js numbers rather than cgal -- we need to add exact support to segments at some point.